aptos-move/framework/aptos-framework/sources/transaction_validation.spec.move

```diff
@@ -0,0 +1,116 @@
+spec aptos_framework::transaction_validation {
+    spec module {
+        pragma verify = true;
+        // pragma aborts_if_is_strict;
+    }
+
+    /// Ensure caller is admin.
+    /// Aborts if TransactionValidation already exists.
+    spec initialize(
+        aptos_framework: &signer,
+        script_prologue_name: vector<u8>,
+        module_prologue_name: vector<u8>,
+        multi_agent_prologue_name: vector<u8>,
+        user_epilogue_name: vector<u8>,
+   ) {
+        use std::signer;
+        let addr = signer::address_of(aptos_framework);
+        aborts_if !system_addresses::is_aptos_framework_address(addr);
+        aborts_if exists<TransactionValidation>(addr);
+   }
+
+    /// Create a schema to reuse some code.
+    /// Give some constraints that may abort according to the conditions.
+    spec schema PrologueCommon {
```

Comment:
Rename this to `PrologueCommonAbortsIf`.

Reply:
The modification is complete.

Reply:
Done.